README.md

```diff
@@ -12,9 +12,7 @@ If you already have Node on your system (we recommend version 17), you can
 install the Node deps like so:
 
 ```
-$ npm install --legacy-peer-deps && \
-    (cd ./release/app/ && npm install --legacy-peer-deps) && \
-    npm install --legacy-peer-deps
+$ npm install --legacy-peer-deps
```

Comment:
do we need the `--legacy-peer-deps` anymore? (`npm i` worked for me on my not-clean box)